tmk_core/common/host_driver.h

```diff
@@ -29,4 +29,5 @@ typedef struct {
     void (*send_mouse)(report_mouse_t *);
     void (*send_system)(uint16_t);
     void (*send_consumer)(uint16_t);
+    void (*send_digitizer)(report_digitizer_t*);
```

Comment:
Is there a particular reason for this?

Reply:
I was trying to stay consistent with the existing way of separating the hw-independent way of building of the report from the hw-dependent way of sending it. Do you have a suggestion for a better way to do this? I know the joystick sending is not made that way, but I think it's a mistake to mix ways of doing the same thing. And I'm allowed to criticize, I made said mistake! :D

Reply:
I think I want to move away from using `host_driver_t`, since even with the Digitizer feature disabled, the (null) function pointer will probably still take up a couple extra bytes. Of course this is also true for the mouse and extrakeys stuff, but they are quite often enabled so I don't see it as that much of an issue.

Reply:
Agreed. Should I use a function pointer outside of the host_driver_t conditioned on DIGITIZER_ENABLE or do you have another suggestion.